lib/matplotlib/backend_bases.py

```diff
@@ -605,13 +606,12 @@ def get_text_width_height_descent(self, s, prop, ismath):
         to the baseline), in display coords, of the string *s* with
         `.FontProperties` *prop*.
         """
+        fontsize = prop.get_size_in_points()
```

Comment:
I guess this was defined multiple times as the second return does not depend on it. But with the very simple function, it should not be a factor.

The similar change in textpath has no potential drawback at all though.

Reply:
There is actually a similar case in `RendererBase.get_text_width_height_descent`, so _if_ you are updating the PR for some reason, feel free to change that as well.

Reply:
both returns depend on fontsize?

Reply:
Not this:
```
        if ismath:
            dims = self._text2path.mathtext_parser.parse(s, dpi, prop)
            return dims[0:3]  # return width, height, descent
```

but no big deal.